tools/ci/cirrus_wheels.yml

```diff
@@ -101,6 +103,7 @@ macosx_arm64_task:
 ######################################################################
 
 wheels_upload_task:
+  use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
```

Comment:
I don't think this is necessary here, because it depends on the wheels being made, and they won't be made unless `CIRRUS_USER_COLLABORATOR == True`.